### ISSUE TITLE
Significantly different validation accuracy depending on platform in "image_classification_from_scratch.py"

### ISSUE THREAD

**Issue by groeneveld** [keras-team/keras-io#268]
In this tutorial after the first epoch val_accuracy is listed as 0.6117
When I copy and paste the same code and run it on Kaggle(TF2.3.0) with the same data, val_accuracy after first epoch is 0.72
When I copy and paste the same code and run it on my machine (TF2.3.1 MacOS 10.15.6) with the same data, val_accuracy after first epoch is 0.54
I repeated these tests all multiple times so it's not a result of random image rotation or flipping.

Shouldn't they all be the same?

I also noticed that on my mac, even after running the malformed file cleaning script, there are still a few file problems that makes the training skip a sample or batch. "Corrupt JPEG data: 128 extraneous bytes before marker 0xd9" or "Warning: unknown JFIF revision number 0.00"

The only deviation from the original code was adding a shape to input: "inputs = keras.Input(shape=(180,180,3))"
